virtual-desktop/nodeServer/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "com.rs.mvd.ng2desktop.server",
```

Comment:
Future: move this out to zlux-proxy-server... do not want services in desktop for cleanliness.